etna/search/views.py

```diff
@@ -580,22 +580,11 @@ def get_datalayer_data(self, request: HttpRequest) -> Dict[str, Any]:
         return data
 
 
-class CatalogueSearchView(BucketsMixin, BaseFilteredSearchView):
-    api_method_name = "search"
-    api_stream = Stream.EVIDENTIAL
-    bucket_list = CATALOGUE_BUCKETS
-    default_group = "tna"
-    form_class = CatalogueSearchForm
-    template_name = "search/catalogue_search.html"
-    search_tab = SearchTabs.CATALOGUE.value
-
-
-class CatalogueSearchLongFilterView(BaseFilteredSearchView):
-    api_method_name = "search"
-    api_stream = Stream.EVIDENTIAL
-    default_group = "tna"
-    form_class = CatalogueSearchForm
-    template_name = "search/catalogue_search_long_filter_chooser.html"
+class BaseLongFilteredSearchView(BaseFilteredSearchView):
```

Comment:
Optional: This naming is not bad, but I would suggest ditching 'Search' from the name (that's not really it's 'function') and try to better clarify what the 'function' of the view is. How do you feel about `BaseLongFilterOptionsView`?

Reply:
Thanks and agreed on `BaseLongFilterOptionsView` . Also looking with the template `long_filter_options.html` seems clearer.